README.md

```diff
@@ -154,6 +154,7 @@ Please visit [Roadmap](https://github.com/lyft/amundsen/blob/master/docs/roadmap
 - [Linux Foundation AI Foundation Landscape](https://landscape.lfai.foundation/)
 - [Lyft’s Amundsen: Data-Discovery with Built-In Trust](https://thenewstack.io/lyfts-amundsen-data-discovery-with-built-in-trust/)
 - [How to find and organize your data from the command-line](https://towardsdatascience.com/how-to-find-and-organize-your-data-from-the-command-line-852a4042b2be)
+- [Facilitating Data discovery with Apache Atlas and Amundsen](https://medium.com/wbaa/facilitating-data-discovery-with-apache-atlas-and-amundsen-631baa287c8b)
```

Comment:
hey @mgorsk1 , could we move the post to https://github.com/lyft/amundsen#blog-posts-and-interviews section ? thanks a lot!

Reply:
I have updated it. Thanks again!